src/csv_overrides.py

```diff
@@ -1,8 +1,12 @@
+from talon import Context, Module, actions, fs, app
 from typing import Optional
-from .conventions import get_cursorless_list_name
-from talon import Context, Module, actions, fs, app, settings
 from datetime import datetime
 from pathlib import Path
+from .conventions import get_cursorless_list_name
+
+SPOKEN_FORM_HEADER = "Spoken form"
+CURSORLESS_IDENTIFIER_HEADER = "Cursorless identifier"
+VSCODE_COMMAND_HEADER = "Vscode command"
```

Comment:
Tempted to move this one into `actions_custom.py`, but I don't feel strongly.  Also might change capitalization to `"VSCode command"`